ansible_collections/arista/avd/plugins/action/anta_runner.py

```diff
@@ -0,0 +1,35 @@
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+from ansible.errors import AnsibleActionFail
+from ansible.plugins.action import ActionBase
+
+from ansible_collections.arista.avd.plugins.plugin_utils.eos_validate_state_utils import get_anta_results
+
+
+class ActionModule(ActionBase):
+    def run(self, tmp=None, task_vars=None):
+        if task_vars is None:
+            task_vars = {}
+
+        result = super().run(tmp, task_vars)
+        del tmp  # tmp no longer has any effect
+
+        ansible_connection = self._connection
+        anta_logging = self._task.args.get("anta_logging")
+        skipped_tests = self._task.args.get("skipped_tests")
+
+        try:
+            anta_results = get_anta_results(vars=dict(task_vars), connection=ansible_connection, logging=anta_logging, skipped_tests=skipped_tests)
+        except Exception as error:
+            raise AnsibleActionFail(message=str(error)) from error
+
+        result["changed"] = True
```

Comment:
Actually nothing changes on the devices. So this should probably be False or unset.